.github/workflows/dependabot-batch-updater.yml

```diff
@@ -0,0 +1,19 @@
+name: Check if dependencies can be updated and create a pull request with updated ones.
+on:
+  schedule:
+    # Run once a week at 5:00
+    - cron:  '0 5 * * */7'
+
+jobs:
+  update_dependencies:
+    runs-on: ubuntu-20.04
+    name: Update dependencies
+    steps:
+      - uses: rasaHQ/dependabot-batch-updater@main
+        with:
+          username: ""
```

Comment:
what is this option for?

Reply:
In the GH action you can define a default value, e.g.

```
  username:
    description: 'Username to authenticate the request'
    required: true
    default: ''
```

Reply:
👍 will do that